### ISSUE TITLE
Metrics "Under Filesystem Read" does not update

### ISSUE THREAD

**Issue by apc999** [Alluxio/alluxio#9442]
**Alluxio Version:**
v2.0.0

**Describe the bug**

**To Reproduce**
```bash
$ bin/alluxio-start.sh local -f
$ bin/alluxio fsadmin report metrics
Total IO: 
    Short-circuit Read                                         0B
    Short-circuit Read (Domain Socket)                         0B
    From Remote Instances                                      0B
    Under Filesystem Read                                      0B
    Alluxio Write                                              0B
    Alluxio Write (Domain Socket)                              0B
    Under Filesystem Write                                     0B

Total IO Throughput (Last Minute): 
    Short-circuit Read                                         0B
    Short-circuit Read (Domain Socket)                         0B
    From Remote Instances                                      0B
    Under Filesystem Read                                      0B
    Alluxio Write                                              0B
    Alluxio Write (Domain Socket)                              0B
    Under Filesystem Write                                     0B

$ bin/alluxio fs cat /sample_tweets_1m.csv > /dev/null

$ bin/alluxio fsadmin report metrics
Total IO: 
    Short-circuit Read                                         0B
    Short-circuit Read (Domain Socket)                         0B
    From Remote Instances                                      0B
    Under Filesystem Read                                      0B
    Alluxio Write                                              0B
    Alluxio Write (Domain Socket)                              0B
    Under Filesystem Write                                     0B

Total IO Throughput (Last Minute): 
    Short-circuit Read                                         0B
    Short-circuit Read (Domain Socket)                         0B
    From Remote Instances                                      0B
    Under Filesystem Read                                171.72KB
    Alluxio Write                                              0B
    Alluxio Write (Domain Socket)                              0B
    Under Filesystem Write                                     0B
```
**Expected behavior**

The total IO of Under Filesystem Read is never incremeted

**Urgency**
Median

**Additional context**

Similar issue: #9354

**Comment by zrss** [comment]
cc

**Comment by calvinjia** [comment]
@zrss The issue should be fixed with #9652 , this fix will be included in the next patch release, 2.0.1
